openslides_backend/presenter/presenter_interface.py

```diff
@@ -18,6 +18,6 @@ class Presenter(Protocol):  # pragma: no cover
     """
 
     def handle_request(
-        self, payload: Payload, logging: LoggingModule, services: Services,
+        self, payload: Payload, logging: LoggingModule, services: Services, user_id: str
```

Comment:
See above: Dependency injection here or at init level?

Reply:
At `init`-level (at least for now).